### PR TITLE
chore(deps): move devDependencies from packages to root

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
chore, deps

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
NA
**Summary**
move devDependencies from packages to root

**Does this PR introduce a breaking change?**
no

**Other information**
no

### CODE REVIEW

File: packages/serve/package.json [hunk 1]
```diff
@@ -9,9 +9,6 @@
     "access": "public"
   },
   "license": "MIT",
-  "devDependencies": {
-    "webpack-dev-server": "3.10.3"
```
Comment (anshumanv) [inline packages/serve/package.json:13]: I think we can let this stay here, let's only move the ones used in >1 package

Comment (snitin315) [reply packages/serve/package.json:13]: only `serve` and `package-utils` have `devDependencies` field in `package.json`.

File: packages/package-utils/package.json [hunk 2]
```diff
@@ -39,10 +39,6 @@
     "execa": "^4.0.0",
     "global-modules": "^2.0.0"
   },
-  "devDependencies": {
-    "@types/cross-spawn": "^6.0.1",
-    "@webpack-cli/package-utils": "^1.0.1-rc.1"
-  },
```
Comment (snitin315) [inline packages/package-utils/package.json:45]: Don't know why `@webpack-cli/package-utils` is a dev dep of `@webpack-cli/package-utils` 😄

File: packages/serve/package.json (hunk 1 above)
Comment (rishabh3112) [reply packages/serve/package.json:13]: We should move all to root imo, this way we would manage all from same place.

File: packages/package-utils/package.json (hunk 2 above)
Comment (rishabh3112) [reply packages/package-utils/package.json:45]: Interesting 😄

**Review by evilebottnawi — DISMISSED** [review]
Good job!

File: packages/serve/package.json (hunk 1 above)
Comment (anshumanv) [reply packages/serve/package.json:13]: Sure it's dev dep so doesn't matter

**Review by anshumanv — DISMISSED** [review]

**Comment by evilebottnawi** [comment]
watch tests are flaky need focus on fix them

**Comment by snitin315** [comment]
> watch tests are flaky need focus on fixing them

@rishabh3112 has created a PR for the same #1890

**Review by evilebottnawi — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Pull request merged**